local-test-infra/local-pod-under-test.yaml

```diff
@@ -28,3 +28,14 @@ spec:
             limits:
               memory: 512Mi
               cpu: 0.25
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: test
+                operator: In
+                values:
+                - test
+            topologyKey: "kubernetes.io/hostname"
```

Comment:
should this be hostname or node?

Reply:
from my understanding, key can be any term, e.g, Zone, hostname, node, rack or power